### ISSUE TITLE
"libcublas.so.10: cannot open shared object file: No such file or directory"

### ISSUE THREAD

**Issue by thaldral** [muellerzr/fastinference#16]
When running `from fastinference.inference import *` a error is thrown:

> "OSError: libcublas.so.10: cannot open shared object file: No such file or directory"

The package was installed with `$ pip install fastinference`.

When running `$ pip show fastinference` this is the output:

```

Name: fastinference
Version: 0.0.25
Summary: A collection of inference modules
Home-page: https://github.com/muellerzr/fastinference/tree/master/
Author: Zachary Mueller
Author-email: muellerzr@gmail.com
License: Apache Software License 2.0
Location: /home/lucas/anaconda3/lib/python3.7/site-packages
Requires: fastai
```

Running `nvidia-smi` returns:

```

CUDA Version: 11.0

```

**Comment by muellerzr** [comment]
For now, I would recommend installing CUDA 10 if possible, otherwise I'll be looking into it in the meantime

**Comment by thaldral** [comment]
Hi, Zach. Thanks for the repply.

I discovered some issues with my Anaconda as a whole. I decided to delete it this morning and downloaded/installed Miniconda. I created a new environment from scratch with it and cloned your the classes repository (Pratical Deep Learning).

Tried to run again installing the newest version of fastai.

`$ pip install fastai - U`

From `pip show fastai` I got:
```
Name: fastai
Version: 2.0.13
```

I installed fastainference again. Checked GPU name with a torch command, and everything was ok. I could import fastinference normally. However, I tried to get a predict from a learner that I loaded before the import (I'm coding with Jupyter) and got a new error and creating a new issue to not confuse. Thanks.
